pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -130,7 +130,7 @@ func SplitDirs(dirs string) []string {
 }
 
 // ProbeNetworkPlugins : get the network plugin based on cni conf file and bin file
-func ProbeNetworkPlugins(confDir, cacheDir string, binDirs []string) []network.NetworkPlugin {
+func ProbeNetworkPlugins(confDir, cacheDir string, binDirs []string) []network.Plugin {
```

Comment:
Fixing this makes the code less readable imo and it'll be good to leave this as is. What do you think?

Reply:
Same as me